plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -933,7 +937,7 @@ private PlugNicAnswer execute(PlugNicCommand cmd) {
              */
```

Comment:
Not directly related to the PR but I've been seening the discussion and I've also seen that you've cleaned up a lot of the code @mike-tutkowski. is it safe to remove this commented block also?